irohad/consensus/yac/yac_peer_orderer.hpp

```diff
@@ -22,6 +22,10 @@
 #include "consensus/yac/cluster_order.hpp"
 #include "consensus/yac/yac_hash_provider.hpp"
 
+//TODO: Please review as to whether it is really necessary to have both a YacPeerOrderer and a ClusterOrdering.
```

Comment:
ClusterOrdering is data - it stores the order and provides accessor functions. YacPeerOrderer is the interface for retrieving the order (possibly including database access and permutation), which returns ClusterOrdering objects.